src/image_config.rs

```diff
@@ -5,106 +5,406 @@ use std::collections::BTreeMap;
 lazy_static! {
     pub static ref IMAGE_MAP: BTreeMap<(u32, u32, u32), &'static str> = {
         let mut m = BTreeMap::new();
-        m.insert((1, 14, 2), "sha256:d4bcffdb9403b154ceeab3063cc5a846a0b4d59a41a8b708249fcc4239b4129d");
-        m.insert((1, 14, 3), "sha256:4adf5dfa0812d3b6835f65fb19687d705d057b7e7cb0cfe14abd240679e12de2");
-        m.insert((1, 14, 4), "sha256:67cf2a1be16075ca0b53325212c03f48624ca68cdd86dcaaf5a06ff18e959fae");
-        m.insert((1, 14, 5), "sha256:608c50412eb0459b0c894117c1e468a480a6f4ea38c4c074e85fc565ecd29214");
-        m.insert((1, 14, 6), "sha256:167ed207752e2ec495851e096bf56c1a276b3330e754b322d401c1087a31f540");
-        m.insert((1, 14, 7), "sha256:b73ccf1e636e278e20c5789b3d3be3e4a72e7813350a93e2ef39a66933175272");
-        m.insert((1, 14, 8), "sha256:a1f76a938980f4bd05ac92829660a781cab167a9a8dd2179787cf23a17526502");
-        m.insert((1, 14, 9), "sha256:073eb70e7fa3be8c7d7809e03bdedcf7ddd8744c8004921151c7b1c19fe760fd");
-        m.insert((1, 14, 11), "sha256:e82b7df2bb23e6031a04b99e361acb0340b86857e9ad6c76db61fe65757166fd");
-        m.insert((1, 14, 12), "sha256:15341a61fcee68cd04f0b0b566e8c2a00978899b3d98aa34e4a30bfac7d6deea");
-        m.insert((1, 14, 13), "sha256:4976bd71a0bfa868c28c456bfb080ac59719c570ca6eb0917b445851c4615af4");
-        m.insert((1, 14, 14), "sha256:2700fb6433143e6e8c66ba3dffbad50f001e80c1b4e86815f94b32323f05332d");
-        m.insert((1, 14, 15), "sha256:0cf3a1945d24f2525e165ddef8b173d8a4387563477fbbae6696c15f4c51e26e");
-        m.insert((1, 14, 16), "sha256:c6d3797b4a7790163049933c361ea2a74b4dff8d6a42434e6a04d0d82c04eda0");
-        m.insert((1, 14, 17), "sha256:2aa6adc2a856df8977a332f089ab352211eb28812823121e3b5aedd6bbb89df1");
-        m.insert((1, 14, 18), "sha256:fe75071a171f817799cb33821f7d680d594d6f066440c9a4c3fd54b55b681439");
-        m.insert((1, 14, 19), "sha256:3e567c978dfc782f60c4ce768738cfacc5c36bd005becd1c14379b120d8f9174");
-        m.insert((1, 14, 20), "sha256:e9501544d3d085bbbff7c100898ce29bf085dd8c851fb85a73b00f72b2ce7c4c");
-        m.insert((1, 14, 21), "sha256:1531e719c4b2675b59b4be410b4cb8d2ecfaf337b6e111dcaf986cf7c343b7e3");
-        m.insert((1, 14, 22), "sha256:e6d6f1f095de3188bfcf9356f91102b85290ffeba6388debe11706e1d9a02826");
-        m.insert((1, 14, 23), "sha256:f6193af3b4e86a5a40c77b6d4294fccb914c938ef16d8db1d67c7a571797345e");
-        m.insert((1, 14, 24), "sha256:866488d56ec3f1a823cb9ecd7652effdc8746b96827209b5ea4ec78a1fa9d45b");
-        m.insert((1, 14, 25), "sha256:a274e5245609baa175d2fcca93a6427c9f295cbbca7a705076d5f977db28d360");
-        m.insert((1, 14, 26), "sha256:ee6ca9b4dcdb58413259545f833f3834733bb281790b6778099c1a586d267fa8");
-        m.insert((1, 14, 27), "sha256:66beebc3bd73ad7b0c694dd0f7027f39e211932caecc0d908467bb00c8f83812");
```

Comment:
can you please regenerate this w/o linting so I can check the diffs please?

Reply:
I've regenerated this without linting. Could you please review it?